csss-site/src/events/mountain_madness/static/mountain_madness_static/2023/csss_custom.css

```diff
@@ -0,0 +1,4 @@
+table, th, td {
```

Comment:
where is this used?

Reply:
the schedule is created with a table

Reply:
well your code is not using it so it might not be necessary...